### ISSUE TITLE
Form data file upload request key sent correctly value sent as unidentified

### ISSUE THREAD

**Issue by Ady999999999** [a85/POSTMan-Chrome-Extension#12409]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When using Form Data Content type with the File type selected the request sent will contain the correct key but the value of the key will be unidentified. (Name of the actual file, its reading it when using the UI, when sending the request no)

I have tried in different ways to get the value to show up but it's always unidentified, it's breaking automated tests with newman.

I'm using the Desktop app and the file is placed in the Postman Working Directory, it's an audio file

### Steps To Reproduce

1. Create a New Post Request with Postman Desktop app
2.. Set the Body Content Type to Form Data
3. Set the selection to file, input the Key and the file select a File to Upload that is Present in the Postman Work Directory, be it in a shared location or on a local drive
4. Open the Postman Console
5. Execute the Request
6. Observe in the Console that the Key is sent correctly in the body but the value which should include the file read is set as unidentified

![Postan Issue 1](https://github.com/postmanlabs/postman-app-support/assets/148074818/158d4a6a-b815-48ff-8dff-fac874e63e9f)

![Postman Issue 2](https://github.com/postmanlabs/postman-app-support/assets/148074818/f4a422e8-2653-4484-b58d-9b4ee6af32d8)

### Operating System

Windows

### Postman Version

v10.19

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by giridharvc7** [comment]
I think this is a bug in the console. I tried sending a `.wav` file to an Echo endpoint, Postman seemed to have sent the right file. Can you confirm if the file was actually sent to the endpoint? 
<img width="1512" alt="image" src="https://github.com/postmanlabs/postman-app-support/assets/1040583/9bd2c637-a264-4aeb-9a22-a2eedf37be76">

**Comment by Ady999999999** [comment]
Yes I can confirm the file was actually sent and my endpoint processed it correctly, so yeah might be a console bug.

**Comment by MC-sanderv** [comment]
Closing this issue as resolved.
The file is correctly sent and processed by the endpoint — the unidentified value appears to be a Postman Console display limitation.

Should you have still issues with this on the latest Postman versions, please open a new ticket.
